### ISSUE TITLE
[Bug]: No Data from awb_es_de since last update home assistant

### ISSUE THREAD

**Issue by uluick** [mampfes/hacs_waste_collection_schedule#2651]
### I Have A Problem With:

A specific source

### What's Your Problem

I get no data from this source

### Source (if relevant)

awb_es_de

### Logs

```Shell
2024-09-09 19:04:37.509 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration waste_collection_schedule which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2024-09-09 19:04:42.366 ERROR (SyncWorker_2) [custom_components.waste_collection_schedule.waste_collection_schedule.source_shell] fetch failed for source Abfallwirtschaftsbetrieb Esslingen:
File "/config/custom_components/waste_collection_schedule/waste_collection_schedule/source_shell.py", line 145, in fetch
File "/config/custom_components/waste_collection_schedule/waste_collection_schedule/source/awb_es_de.py", line 56, in fetch
File "/config/custom_components/waste_collection_schedule/waste_collection_schedule/service/ICS.py", line 48, in convert
2024-09-10 01:51:39.248 ERROR (SyncWorker_24) [custom_components.waste_collection_schedule.waste_collection_schedule.source_shell] fetch failed for source Abfallwirtschaftsbetrieb Esslingen:
File "/config/custom_components/waste_collection_schedule/waste_collection_schedule/source_shell.py", line 145, in fetch
File "/config/custom_components/waste_collection_schedule/waste_collection_schedule/source/awb_es_de.py", line 56, in fetch
File "/config/custom_components/waste_collection_schedule/waste_collection_schedule/service/ICS.py", line 48, in convert
```

### Relevant Configuration

```YAML
waste_collection_schedule:
  sources:
    - name: awb_es_de
      args:
        city: right city name
        street: right street name
```

### Checklist Source Error

- [ ] Use the example parameters for your source (often available in the documentation) (don't forget to restart Home Assistant after changing the configuration)
- [X] Checked that the website of your service provider is still working
- [X] Tested my attributes on the service provider website (if possible)
- [X] I have tested with the latest version of the integration (master) (for HACS in the 3 dot menu of the integration click on "Redownload" and choose master as version)

### Checklist Sensor Error

- [X] Checked in the Home Assistant Calendar tab if the event names match the types names (if types argument is used)

### Required

- [X] I have searched past (closed AND opened) issues to see if this bug has already been reported, and it hasn't been.
- [X] I understand that people give their precious time for free, and thus I've done my very best to make this problem as easy as possible to investigate.

**Comment by saschaludwig** [comment]
Most likely related to https://github.com/mampfes/hacs_waste_collection_schedule/issues/2643

**Comment by 5ila5** [comment]
This was caused by a faulty version of icalevents this should hopefully be fixed when updating to the freshly released WCS version 2.2.0

**Comment by saschaludwig** [comment]
Thank you! I can confirm that this is fixed.
